src/grid/Col/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default function Col({ children, col, offset, style }) {
+  const classCol = col && `col-${col}`;
```

Comment:
We should probably still call it column and think of something for offset. In any case, do we need the && here since we have default props?